tests/common/templates/default_acl_rules.json

```diff
@@ -0,0 +1,38 @@
+{
+    "acl": {
+        "acl-sets": {
+            "acl-set": {
+                "dataacl": {
+                    "acl-entries": {
+                        "acl-entry": {
+                            "1": {
+                                "actions": {
+                                    "config": {
+                                        "forwarding-action": "ACCEPT"
+                                    }
+                                },
+                                "config": {
+                                    "sequence-id": 1
+                                },
+                                "l2": {
+                                    "config": {
+                                        "ethertype": "2048",
+                                        "vlan_id": "1000"
+                                    }
+                                },
+                                "input_interface": {
+                                    "interface_ref":
+                                    {
+                                        "config": {
+                                            "interface": "Ethernet12,Ethernet16,Ethernet20,Ethernet24,Ethernet28,Ethernet32,Ethernet36,Ethernet4,Ethernet40,Ethernet44,Ethernet48,Ethernet52,Ethernet56,Ethernet60,Ethernet64,Ethernet68,Ethernet72,Ethernet76,Ethernet8"
```

Comment:
@yutongzhang-microsoft use the same interfaces for different topology?

Reply:
No, this is set by `pre_acl_rules`